tokens/src/lib.rs

```diff
@@ -1744,6 +1749,10 @@ impl<T: Config> fungibles::Inspect<T::AccountId> for Pallet<T> {
 	) -> WithdrawConsequence<Self::Balance> {
 		Self::withdraw_consequence(who, asset_id, amount, &Self::accounts(who, asset_id))
 	}
+
+	fn asset_exists(_asset: Self::AssetId) -> bool {
+		true
```

Comment:
@xlc Do you want to return false here? 
Prevent the substrate from using it later

Reply:
https://github.com/paritytech/substrate/pull/12782

`unimplemented!()` is fine.